src/java.base/share/classes/sun/security/ssl/SSLConfiguration.java

```diff
@@ -108,9 +108,14 @@ final class SSLConfiguration implements Cloneable {
     static final int maxHandshakeMessageSize = GetIntegerAction.privilegedGetProperty(
             "jdk.tls.maxHandshakeMessageSize", 32768);
 
-    // Set the max certificate chain length to 10
-    static final int maxCertificateChainLength = GetIntegerAction.privilegedGetProperty(
-            "jdk.tls.maxCertificateChainLength", 10);
+    // Maximum allowed certificate chain length
+    static final int maxCertificateChainLength;
+
+    // Limit the maximum certificate chain length accepted from clients
+    static final int maxClientCertificateChainLength;
+
+    // Limit the maximum certificate chain length accepted from servers
```

Comment:
I would drop "maximum" and just say "Limit the certificate chain length accepted from servers". It's not worth making this change unless you have to make another change to this file.

Reply:
ok